### PR TITLE
Cherry-pick #15682 to 7.x: AWS Lambda: downgrade Kibana dashboard

### PR DESCRIPTION
**Pull request by mtojek**
Cherry-pick of PR #15682 to 7.x branch. Original message: 

This PR downgrades required Kibana version for AWS Lambda dashboard. It fixes error:

```
$ metricbeat setup --dashboards
Loading dashboards (Kibana must be running and reachable)
Exiting: Failed to import dashboard: Failed to load directory /Users/user/go/src/github.com/elastic/beats/x-pack/metricbeat/kibana/7/dashboard:
  error loading /Users/user/go/src/github.com/elastic/beats/x-pack/metricbeat/kibana/7/dashboard/Metricbeat-aws-lambda-overview.json: returned 422 to import file: <nil>. Response: {"statusCode":422,"error":"Unprocessable Entity","message":"Document \"deab0260-2981-11e9-86eb-a3a07a77f530\" has property \"visualization\" which belongs to a more recent version of Kibana (7.4.2)."}
```

... and fixes `test_dashboards` reported by Travis CI.

### CODE REVIEW

**Comment by mtojek** [comment]
I fixed merge conflicts. Let's wait for the CI to be sure.

File: metricbeat/tests/system/test_base.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def test_dashboards(self):
         )
         exit_code = self.run_beat(extra_args=["setup", "--dashboards"])
 
-        assert exit_code == 0
+        assert exit_code == 0, 'Error output: ' + self.get_log()
```
Comment (jsoriano) [inline metricbeat/tests/system/test_base.py]: Not related, but I guess it doesn't do any harm :slightly_smiling_face:

Comment (mtojek) [reply metricbeat/tests/system/test_base.py]: Actually it helps you finding the reason why the `test_dashboards` step has failed in the CI environment ;)

**Comment by mtojek** [comment]
All checks are green. Resolving.

**Pull request merged**